docs/references/hardware.rst

```diff
@@ -6,3 +6,4 @@ Hardware Supports
    amd.md
    nvidia_jetson.md
    cpu.md
+   ascend_npu.md
```

Comment:
this is now moved to https://github.com/sgl-project/sglang/blob/2449a0afe246d096f58e86c6b5f5563a63598cf4/docs/index.rst?plain=1#L66

Reply:
done